src/features/content-explorer/content-explorer/__tests__/ContentExplorer.test.js

```diff
@@ -78,6 +78,34 @@ describe('features/content-explorer/content-explorer/ContentExplorer', () => {
             expect(wrapper.find('ContentExplorerActionButtons').prop('onSelectedClick')).toEqual(onSelectedClick);
         });
 
+        test('should render ContentExplorerSelectAll with isSelectAllAllowed = true', () => {
+            const isSelectAllAllowed = true;
+            const wrapper = renderComponent({ isSelectAllAllowed });
+
+            expect(wrapper.find('ContentExplorerSelectAll').length).toBe(1);
```

Comment:
You can also use `expect(wrapper.exists('ContentExplorerSelectAll')).toBe(true)`